### PR TITLE
review findings

### PR DESCRIPTION
**Pull request by Hades32**
Plus https://docs.ultimatearcade.io/ start page still talks about Docusaurus

### CODE REVIEW

File: docs/integration/high-level.md [hunk 1]
```diff
@@ -4,24 +4,33 @@ sidebar_position: 2
 
 # High-Level Overview
 
+To integrate with the Ultimate Arcade you need to deliver
+* the game client
+  * a bundle of HTML, JS and assets that runs in the player's browser
+* the game server
+  * a binary (with any needed libraries and assets) that can run on a Linux server
```
[the remark this answers is not in the document]
Comment (Hades32) [reply docs/integration/high-level.md:11]: Yeah, I didn't want to mention Dockerfiles/containers here as most game devs probably don't have experience with them, but maybe there's no way around it... I'll draft something...

File: docs/integration/integration-requirements.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+The litmus test is that given the same seed, and the same user inputs, the exact gameplay simulation will occur every time. For example enemies will always spawn and attack in the exact same way, or a platformer will always generate the map in the same way.
+
+While it does not need to be exactly deterministic, it must be noticeably the same.
+
+## Free to Play/Practice Mode
+
+All games should have a "free to play" or "practice" mode. This is where the player does not pay any tokens up front, but does not earn any tokens.
+
+<!-- TODO: Does this mean there will be no leaderboard for these practice plays? -->
```
Comment (Hades32) [inline docs/integration/integration-requirements.md:45]: @danthegoodman1 is this the case?

File: docs/integration/code-adaptions.mdx [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+    UnityEngine.Random.InitState(AutoConnect.RandomSeed.GetHashCode());
+    // and we should only allow players to join after we setup all randomness
+}
+```
+
+</TabItem>
+<TabItem value="js" label="JavaScript">
+
+```js
+alert("TODO");
```
Comment (Hades32) [inline docs/integration/code-adaptions.mdx:52]: @danthegoodman1 can you fill in the JS examples?

File: docs/integration/integration-requirements.md (hunk 2 above)
Comment (Hades32) [reply docs/integration/integration-requirements.md:43]: Also, this means we must tell the iframed game that it shouldn't try to talk to a server

File: docs/integration/code-adaptions.mdx (hunk 3 above)
Comment (danthegoodman1) [reply docs/integration/code-adaptions.mdx:52]: Will do!

**Comment by danthegoodman1** [comment]
@Hades32 going to merge and do a branch for the JS additions you can review

**Pull request merged**